### ISSUE TITLE
In homepage, the "location" section feels empty

### ISSUE THREAD

**Issue by privat** [pyconca/2017-web#74]
https://github.com/pyconca/2017-web/blob/151c9731c0df767ba06f77321403ad1820a47100/pyconca2017/templates/pages/home.html#L52

1. add a link to the venue page 'https://2017.pycon.ca/venue/'
2. reuse the fist paragraph from the venue page. eg. 'PyCon Canada 2017 is being held at Université du Québec à Montréal from 18-21 November 2017 in Montréal, Québec, Canada. The event is organized in collaboration with the UQAM's Department of Computer Science.'
3. add a picture/photo. eg. https://commons.wikimedia.org/wiki/File:UQAM_(Universit%C3%A9_du_Qu%C3%A9bec_%C3%A0_Montr%C3%A9al)_-_8-2017_-1_m.jpg or you can look for something nice and usable in https://www.google.ca/search?q=uqam&tbm=isch

**Comment by frdeso** [comment]
I agree we should beef up this section. I don't think that a picture would fit there but we need to come up with text there. Maybe mention that the conference is 18-19 and sprints are on 20-21.
